Heatington/Optimizer/Optimizer.cs

```diff
@@ -1,123 +1,210 @@
-using FrancescoDummyClasses;
+using Heatington.Data;
+using Heatington.Models;
 
 namespace Heatington.Optimizer;
 
-public class Optimizer
+public class Opt
 {
-    private List<ProductionUnitFs> _pUnits = new List<ProductionUnitFs>();
-    private List<FrancescoEnergyData> _energyDataEntries = new List<FrancescoEnergyData>();
-    private List<ResultHolder> _resultEntries = new List<ResultHolder>();
+    private List<DataPoint>? _dataPoints = new List<DataPoint>();
+    private List<ProductionUnit> _productionUnits = new List<ProductionUnit>();
+    public List<ResultHolder>? Results { get; private set; }
 
-    public void OptimizerScenario1()
+    public void LoadData()
```

Comment:
What do think about using a try-catch statement here?

Reply:
I am not sure about that as I think it's the sdm's responsibility.